src/index.html

```diff
@@ -7,6 +7,289 @@
+          <div class="projects__slider">
+            <div class="projects__slider__cards">
+
+              <div class="projects-card">
+                <img src="./assets/images/wildstone-infra.png" alt="Wildstone Infra Hotel picture" class="projects-card__img">
+                <div class="projects-card__descr">
+                  <span class="projects-card__descr-title">Wildstone Infra Hotel</span>
+                  <span class="projects-card__descr-address">2715 Ash Dr. San Jose, South Dakota</span>
+                </div>
+              </div>
+
+              <div class="projects-card">
+                <img src="./assets/images/wish-stone.png" alt="Wish Stone Building picture" class="projects-card__img">
+                <div class="projects-card__descr">
+                  <span class="projects-card__descr-title">Wish Stone Building</span>
+                  <span class="projects-card__descr-address">2972 Westheimer Rd. Santa Ana, Illinois</span>
+                </div>
+              </div>
+
+              <div class="projects-card">
+                <img src="./assets/images/parkinston.png" alt="Mr. Parkinston's House picture" class="projects-card__img">
+                <div class="projects-card__descr">
+                  <span class="projects-card__descr-title">Mr. Parkinston&#39;s House</span>
+                  <span class="projects-card__descr-address">3517 W. Gray St. Utica, Pennsylvania</span>
+                </div>
+              </div>
+
+              <div class="projects-card">
+                <img src="./assets/images/oregano-height.png" alt="Oregano Height picture" class="projects-card__img">
+                <div class="projects-card__descr">
+                  <span class="projects-card__descr-title">Oregano Height</span>
+                  <span class="projects-card__descr-address">2464 Royal Ln. Mesa, New Jersey</span>
+                </div>
+              </div>
+
+            </div>
+            <div class="projects__slider__navigation">
+              <button class="description-slider__button btn-next projects__slider-btn">Next
+                <img src="./assets/icons/arrow-next.svg" alt="icon next" class="description-slider__icon icon-next">
+              </button>
+              <div class="slider__navigation__round-tabs"></div>
+              <button class="description-slider__button btn-back projects__slider-btn">Back
+                <img src="./assets/icons/arrow-back.svg" alt="icon next" class="description-slider__icon icon-back">
+              </button>
+            </div>
+          </div>
+        </div>
+      </section>
+<!-- ------------------ FEEDBACK --------------------- -->
+      <section class="feedback">
+        <h2 class="feedback__title">What can us do for you?</h2>
+        <p class="feedback__description">We are ready to work on a project of any complexity, whether it&#39;s commercial or residential.</p>
+        <form action="#" method="send" class="feedback__form">
```

Comment:
Invalid attribute value for the `method`. It should be one of this set: GET, POST, PATCH, PUT, DELET.

Reply:
post,  bcz we are supposed to send data to server stated in action. Am I right?